### PR TITLE
Add talks and presentations section to README

### PR DESCRIPTION
**Pull request by nirs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,14 @@ For each type of obstacle there is a different action, and different points.
 See [examples/README](examples/README) for explanation on how to write a driver module.
 
 
+## Talks and presentations
+
+- [Watch](https://www.youtube.com/watch?v=41oxZr43Ih0) Fred and Ori talk from
```
Comment (rollandf) [inline README.md]: Can you add the full names ?

Comment (nirs) [reply README.md]: Sure

**Comment by nirs** [comment]
@rollandf, @orrabin I added links to your github pages, please review.

**Comment by nirs** [comment]
With single link to the talk:
https://github.com/nirs/ROSE/blob/b7a7c510dfe6fa68fb9bc8290bb666d06936385c/README.md

With links to github pages:
https://github.com/nirs/ROSE/blob/05d9c23ab64cf8ee43a74a66d6c8dd2e7d4302b4/README.md

The version with the single link to the talk looks better.

**Comment by rollandf** [comment]
LGTM

**Pull request merged**